File: Examples/RudderSampleApptvOSObjC/RudderSampleApptvOSObjC/ViewController.m
//
//  ViewController.m
//  Sample-tvOS
//
//  Created by Desu Sai Venkat on 18/10/21.
//

#import "ViewController.h"
#import "AppDelegate.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
}

- (IBAction)identify:(id)sender {
    [AppDelegate identify];
}
- (IBAction)track:(id)sender {
    [AppDelegate track];
}
- (IBAction)reset:(id)sender {
    [AppDelegate reset];
}
- (IBAction)optIn:(id)sender {
    [AppDelegate optIn];
}
- (IBAction)optOut:(id)sender {
    [AppDelegate optOut];
}
- (IBAction)screen:(id)sender {
    [AppDelegate screen];
}

@end

File: RudderSampleProject/RudderSampleProject/ViewController.swift
//
//  ViewController.swift
//  RudderSampleProject
//
//  Created by Arnab Pal on 09/10/19.
//  Copyright © 2019 Arnab Pal. All rights reserved.
//

import UIKit
import RudderSdkCore

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
        
        AppDelegate.rudderClient?.track(message: RudderMessageBuilder()
            .withEventName(eventName: "level_up")
            .withEventProperties(properties: TrackPropertyBuilder()
                .setCategory(category: "test_category")
                .build())
            .withUserId(userId: "test_user_id")
            .build()
        )
    }


}

